common/core/src/commonMain/kotlin/com/trafi/core/ApiResult.kt

```diff
@@ -52,6 +52,26 @@ sealed class ApiResult<out T : Any> {
             val error: com.trafi.core.model.Error? = null,
         ) : Failure<T>(throwable)
 
+        /**
+         * The server returned an error covered by [ErrorCode.Users] types.
+         */
+        class UserError<T : Any>(
+            throwable: Throwable,
+            val httpStatusCode: Int,
+            val code: ErrorCode.Users,
+            val error: com.trafi.core.model.Error? = null,
```

Comment:
`Error` is not nullable in these cases, right?

Reply:
absolutely!